packages/synapse-react-client/src/components/GenericCard/CardUtils.tsx

```diff
@@ -0,0 +1,169 @@
+import { CardLink } from '@/components/CardContainer/CardLink'
+import { isTableEntity } from '@/utils/functions/EntityTypeUtils'
+import { PRODUCTION_ENDPOINT_CONFIG } from '@/utils/functions/getEndpoint'
+import {
+  convertDoiToLink,
+  SYNAPSE_ENTITY_ID_REGEX,
+} from '@/utils/functions/RegularExpressions'
+import {
+  ColumnModel,
+  ColumnType,
+  Entity,
+  FileHandleAssociateType,
+  FileHandleAssociation,
+  SelectColumn,
+} from '@sage-bionetworks/synapse-types'
+import { TargetEnum } from '../CardContainerLogic/CardContainerLogic'
+
+export const getFileHandleAssociation = (
```

Comment:
moved some of these function out of the `TableRowGenericCard` file to improve HMR